crates/router/src/connector/stax/transformers.rs

```diff
@@ -185,21 +332,37 @@ impl TryFrom<types::RefundsResponseRouterData<api::RSync, RefundResponse>>
     fn try_from(
```

Comment:
TryFrom logic needs to be separate because Refunds Execute returns a parent transaction with a child containing the details while RSync just returns a child transaction.